### DISCUSSION TITLE
Support next/prev completion actions still not working for JavaScript and Ruby files.

### DISCUSSION THREAD

**Discussion by robbporto** [community/community#8041] [Copilot Conversations]
In the changelog for the newest version, it says:

> Support next/prev completion actions in all files, not just Java and Python.

I'm trying to loop through the options by typing `alt + ]` but it's not working.

**Comment by jansorg** [comment]
Could you post a snippet where this doesn't work for you?
It's still possible that there's just nothing to show, but the UI isn't showing this so it's not ideal right now.
